syncback/Tiltfile

```diff
@@ -9,7 +9,9 @@ verify_rsync_path = os.path.join(os.getcwd(), 'verify_rsync.sh')
 print('-- syncback extension checking for local rsync --')
 local('{} {}'.format(verify_rsync_path, MIN_LOCAL_RSYNC_VERSION))
 
-def syncback(name, k8s_object, src_dir, paths=None, target_dir='.', container='', namespace='', verbose=False):
+DEFAULT_EXCLUDES = ['.git', '.gitignore', '.dockerignore', 'Dockerfile', '.tiltignore', 'Tiltfile', 'tilt_modules']
```

Comment:
this is probably OK for now, but it probably won't work if you have sub-Tiltfiles. You probably need to make them relative to `config.main_dir`

Reply:
These are paths on the container though / relative to src_dir, so I think it'll be fine!